msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -170,9 +171,7 @@ The final step to running your app in Microsoft 365 and Outlook is to sideload y
 
     :::image type="content" source="images/toolkit-zip-teams-metadata-package.png" alt-text="'The screenshot is an example that shows the Zip Teams metadata package' option in Teams Toolkit extension for Visual Studio Code.":::
 
-1. Sign in to Teams with your sandbox tenant account, and toggle into  *Developer Preview* mode. Select the ellipsis (**...**) menu by your user profile, then select: **About** > **Developer preview**.
```

Comment:
@erikadoyle - similiar to other tutorial, pretty sure we want to remove this given https://github.com/MicrosoftDocs/msteams-docs/pull/7930 right?